ci/tasks/prepare-perftests-codebuild.yml

```diff
@@ -0,0 +1,94 @@
+---
+platform: linux
+image_resource:
+  type: registry-image
+  source:
+    repository: alpine
+    tag: 3.16
+outputs:
+  - name: run-codebuild-configuration
+run:
+  path: ash
+  args:
```

Comment:
Note, this script hasn't changed at all, just moved from in the pipeline yaml into this task